javaCalc/src/Init.java

```diff
@@ -0,0 +1,14 @@
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
+
+public class Init {
+	
+	public static JButton makeButton(String text, int height, int width) {

```

Comment:
Static is generally a big nope, objects that want this behaviour should just inject an instance of this class into themselves instead.